src/Physics/Plugins/babylon.cannonJSPlugin.ts

```diff
@@ -116,7 +116,7 @@
                                 this.removePhysicsBody(childImpostor);
                                 childImpostor.physicsBody = null;
                             }
-                            childImpostor.parent = mainImpostor;
+                            (childImpostor as any).parent = mainImpostor;

```

Comment:
well, PhysicImpostor.parent doesn't have a setter! so unless the property setter is added at runtime, it's a bug! :)